sky/spot/spot_state.py

```diff
@@ -22,9 +22,12 @@
 _CONN = sqlite3.connect(str(_DB_PATH))
 _CURSOR = _CONN.cursor()
 
-# `spot` table contains all the finest-grained jobs, including all the
-# sub jobs of a spot job. All sub jobs will have the same `new_job_id`.
-# The `job_name` is the name of the sub job.
+# `spot` table contains all the finest-grained tasks, including all the
+# tasks of a spot job. All tasks will have the same `spot_job_id`.
+# The `job_name` is the name of the task (naming of that column is
+# legacy issue and should be changed to `task_name`).
+# The `job_id` is now not really a job id, but a only a unique identifier
+# for all the tasks. We will use `spot_job_id` to identify the spot job.
```

Comment:
Maybe we should add more comment here on old `spot` tables before this schema change. Something like L25-30 above only apply to new jobs submitted after this schema change, and old jobs will have their original meanings...

Reply:
I think it should be fine as all the comments here can apply to the old jobs as well, since we have copied those columns?